packages/expo-ui/ios/List.swift

```diff
@@ -0,0 +1,122 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import SwiftUI
+
+class ListProps: ExpoSwiftUI.ViewProps {
+  @Field var listStyle: String = "automatic"
+  @Field var moveEnabled: Bool = false
+  @Field var deleteEnabled: Bool = false
+  @Field var selectEnabled: Bool = false
+  @Field var scrollEnabled: Bool = false
+  @Field var editModeEnabled: Bool = false
+  var onDeleteItem = EventDispatcher()
+  var onMoveItem = EventDispatcher()
+  var onSelectionChange = EventDispatcher()
+}
+
+struct ListView: ExpoSwiftUI.View {
+  @EnvironmentObject var props: ListProps
+  @State private var selection: Set<Int> = []
+  @State var editModeEnabled: EditMode = .inactive
+  @State var search: String = ""
+  var body: some View {
+    let list = List(selection: props.selectEnabled ? $selection : nil) {
+      UnwrappedChildren { child, isHostingView in
+        child
+          .if(!isHostingView) {
+            $0.offset(
+              x: UIDevice.current.userInterfaceIdiom == .pad
+              ? IPAD_OFFSET : IPHONE_OFFSET)
+          }
+      }
+      .onDelete(perform: handleDelete)
+      .onMove(perform: handleMove)
+      .deleteDisabled(!props.deleteEnabled)
+      .moveDisabled(!props.moveEnabled)
+    }
+      .modifier(ListStyleModifer(style: props.listStyle))
+      .onAppear {
+        editModeEnabled = props.editModeEnabled ? .active : .inactive
+      }
+      .onChange(of: props.editModeEnabled) { newValue in
+        withAnimation {
+          editModeEnabled = newValue ? .active : .inactive
+        }
+      }
+      .onChange(of: selection) { selection in
+        print(selection)
+        handleSelectionChange(selection: selection)
+      }
+      .modifier(ScrollDisabledModifier(scrollEnabled: props.selectEnabled))
+      .environment(\.editMode, $editModeEnabled)
+    if #available(iOS 16.0, tvOS 16.0, *) {
+      list.scrollDisabled(!props.scrollEnabled)
+    } else {
+      list
+    }
```

Comment:
In `List/index.tsx` the default for `scrollEnabled` is `false`, but since the native property is called `scrollDisabled` I'd assume that for iOS < 16.0 the scroll is always enabled. If this is the case I think the default for `scrollEnabled` should be `true` on our side.

Reply:
Agreed!